src/ensembl/src/content/app/species-selector/state/speciesSelectorActions.ts

```diff
@@ -0,0 +1,100 @@
+import { createStandardAction, createAsyncAction } from 'typesafe-actions';
+
+// import apiService from 'src/services/api-service';
```

Comment:
Is this commented out because mock data is used?

Reply:
Yep.